Makefile

```diff
@@ -125,42 +137,40 @@ FATFS_SRC       = $(notdir $(wildcard $(FATFS_DIR)/*.c))
 
 CSOURCES        := $(shell find $(SRC_DIR) -name '*.c')
 
-
 ifeq ($(TARGET),$(filter $(TARGET),$(F3_TARGETS)))
 # F3 TARGETS
 
 STDPERIPH_DIR   = $(ROOT)/lib/main/STM32F30x_StdPeriph_Driver
 STDPERIPH_SRC   = $(notdir $(wildcard $(STDPERIPH_DIR)/src/*.c))
 EXCLUDES        = stm32f30x_crc.c \
-                  stm32f30x_can.c
+				  stm32f30x_can.c
```

Comment:
Its just github (expanding tabs out further than it should). Unfortunately the replacing of tabs with spaces upsets "make". Otherwise I'd opt for 4 spaces instead of tabs.